### DISCUSSION TITLE
where is the authorization key stored?

### DISCUSSION THREAD

**Discussion by antsy** [community/community#7352] [Copilot Conversations] [answered]
I accidentally tried to authorize Copilot with my Work Github account when my invite was for my Personal Github account, I can't find where VSCode extension settings are and re-installing won't allow me to re-authorize?

**Comment by RayburnTrading** [comment]
I also have the same issue, originally authenticated with a personal Github account and now I've authenticated with the correct work account it says I am not authorised!?

**Comment by antsy** [comment, marked as answer]
Ok I found solution from another thread. You need to log out in VSCode, it's done via the "Account" button at bottom left corner of VSCode 👍
